src/middleware/queriesV2.ts

```diff
@@ -259,10 +259,12 @@ function createEventFilters(attributes: QueryArgs<Event, EventArgs>): {
 } {
   const args: string[] = ['$start: Int', '$size: Int'];
   const filters: string[] = [];
+  const typeMap: Record<string, string> = { moduleId: 'ModuleIdEnum', eventId: 'EventIdEnum' };
```

Comment:
These `createXFilter` functions are similar enough that it merits using a single `createQueryParams` or `createArgsAndFilters` function which takes the attributes and the type map as parameters. If you need to call the function more than once with the same type map, just assign it to a variable (or curry it if you want)